docs/IE_DG/supported_plugins/AutoPlugin_Debugging.md

```diff
@@ -0,0 +1,136 @@
+# Debugging Auto-Device Plugin {#openvino_docs_IE_DG_supported_plugins_AUTO_debugging}
+
+## Debug log
+In case of execution problems, just like all other plugins, Auto-Device provides the user with information on exceptions and error values. If the returned data is not enough for debugging purposes, more information may be acquired by means of LOG_LEVEL.
+
+There are six levels of logs, which can be called explicitly or set via the OPENVINO_LOG_LEVEL  environment variable (can be overwritten by compile_model or set_config):  
+0. LOG_NONE 
+1. LOG_ERROR 
+2. LOG_WARNING
+3. LOG_INFO
+4. LOG_DEBUG
+5. LOG_TRACE
+
+@sphinxdirective
+.. tab:: C++ API
+
+   .. code-block:: cpp
+
+      ov::runtime::Core core
+      
+      // read a network in IR, PaddlePaddle, or ONNX format
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
+      
+      // load a network to AUTO and set log level to debug
+      ov::runtime::CompiledModel compiled_model = core.compile_model(model, “AUTO”, {{"LOG_LEVEL ", "LOG_DEBUG"}});
+      
+      // or set log level with set_config and load network
+      core.set_config({"LOG_LEVEL", "LOG_DEBUG"}}, "AUTO");
+      ov::runtime::CompiledModel compiled_model = core.compile_model(model, “AUTO”);
+	  
+.. tab:: Python API
+
+   .. code-block:: python
+
+      from openvino.runtime import Core
+      core = Core()
+      
+      # read a network in IR, PaddlePaddle, or ONNX format
+      model = core.read_model(model_path)
+      
+      # load a network to AUTO and set log level to debug
+      compiled_model = core.compile_model(model=model, device_name="AUTO", config={"LOG_LEVEL":"LOG_DEBUG"});
+      
+      // or set log level with set_config and load network
+      ie.SetConfig(config={"LOG_LEVEL":"LOG_DEBUG"}, device_name="AUTO");
```

Comment:
`SetConfig` => `set_property`

Reply:
Thanks! I will point it out to the Auto team.